.github/workflows/native-shared-library.yml

```
@@ -0,0 +1,42 @@
+name: native-shared-library
+on:
+  push:
+    paths:
+      - 'native-shared-library/**'
+      - '.github/workflows/native-shared-library.yml'
+  pull_request:
+    paths:
+      - 'native-shared-library/**'
+      - '.github/workflows/native-shared-library.yml'
+  schedule:
+    - cron: "0 0 1 * *" # run every month
+  workflow_dispatch:
+permissions:
+  contents: read
+  pull-requests: write # for `native-image-pr-reports` option
+jobs:
+  run:
+    name: Run 'native-shared-library'
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        version: [latest, dev]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          version: ${{ matrix.version }}
+          java-version: '19'
+          components: 'native-image'
```

```suggestion
          components: 'native-image,llvm-toolchain'
```

and remove ` gu install llvm-toolchain` below.